### PR TITLE
Patch 1

### PR DESCRIPTION
**Pull request by codeharmonics**

### CODE REVIEW

**Comment by codeharmonics** [comment]
The goal here is to provide links to style guides. We structured it so that readers could find the guide they need. 

Should be ready to go. Text is complete and proofread but there is a need to add more links. We have provided over 20 good links to style guides.

**Comment by codeharmonics** [comment]
This is the current version. ignore https://github.com/writethedocs/www/pull/291

**Comment by ericholscher** [comment]
Thanks! 💯 Will read this over after the conference, and get it merged!

File: docs/guide/writing/style-guides.md [hunk 1]
```diff
@@ -0,0 +1,91 @@
...
+There's science behind this. Our brains appear to be hardwired to identify differences -- anything that stands out from its context will catch our attention. A lot of variation will drain our cognitive resources, making it more difficult for us to assimilate information. The degree to which cognitive resources are drained varies between people, but in general it is a good idea to minimize the [cognitive load](https://en.wikipedia.org/wiki/Cognitive_load) of your communications.
+
+## Different styles for different folk
+
+Choosing the right style guide for your project depends on your particular context:
+  - Who you are
+  - Who you are writing for
+  - What you are writing about
+
+> If you really need to pick a style guide ***right now***, we recommend you take a look at the [Google Developer Documentation Style Guide](https://developers.google.com/style/)
```
Comment (plaindocs) [inline docs/guide/writing/style-guides.md]: I'm not sure this is the one we want to recommend as tl;dr

File: docs/guide/writing/style-guides.md [hunk 2]
```diff
@@ -0,0 +1,91 @@
...
+- [Open SUSE Style Guide](https://doc.opensuse.org/products/opensuse/Styleguide/opensuse_documentation_styleguide_sd/)
+- [gnome Style Guide](https://developer.gnome.org/gdp-style-guide/2.32/gdp-style-guide.html)
+- [Department of Defense Writing Style Guide and Preferred Usage](http://www.esd.whs.mil/DD/)
+
+### Selecting a good style guide for your audience
+
+Technical documentation takes many forms, each one targeting a certain need. Here, we can think of examples like tutorials, API documentation, and user manuals. So while many style guides adequately cover the main concerns of all technical documentation, sometimes a more specialized guide is needed.
+
+#### User Guide / Manuals
+
```
Comment (plaindocs) [inline docs/guide/writing/style-guides.md]: maybe explicitly state "more info needed" on these empty headers?

File: docs/guide/writing/style-guides.md [hunk 3]
```diff
@@ -0,0 +1,91 @@
...
+- [google snippets](https://developers.google.com/style/code-in-text)
+- [Ruby / other languages](http://guides.rubyonrails.org/api_documentation_guidelines.html)
+- [Al language snippets](https://bocoup.com/blog/documenting-your-api)
+
+#### Release Notes
+
+#### API documentation
+
+- https://developers.google.com/style/api-reference-comments
```
Comment (plaindocs) [inline docs/guide/writing/style-guides.md]: Linkify this link?

File: docs/guide/writing/style-guides.md [hunk 4]
```diff
@@ -0,0 +1,91 @@
...
+## Writing Style
+The style guides below focus on the actual writing craft itself. They consider how to make technical content readable, clear, succinct, and engaging. 
+
+- [The Sense of Style](https://stevenpinker.com/publications/sense-style-thinking-persons-guide-writing-21st-century)
+- [Stylish Academic Writing](http://www.hup.harvard.edu/catalog.php?isbn=9780674064485)
+- [Voice and Tone](https://styleguide.mailchimp.com/)
+- [Writing Style](https://developers.google.com/style/)
+
+## Other resources
+If you don't think that any of these resources meet your needs, maybe you should consider creating your [own style guide document](http://www.writethedocs.org/style-guide/).
```
Comment (plaindocs) [inline docs/guide/writing/style-guides.md]: This link seems a little redundant?

**Review by plaindocs — CHANGES_REQUESTED** [review]
Couple of picky change requests

File: docs/guide/writing/style-guides.md (hunk 1 above)
Comment (Bradamant3) [reply docs/guide/writing/style-guides.md]: Agreed

File: docs/guide/writing/style-guides.md (hunk 3 above)
Comment (Bradamant3) [reply docs/guide/writing/style-guides.md]: Also it's from the google guide ...

File: docs/guide/writing/style-guides.md (hunk 4 above)
Comment (Bradamant3) [reply docs/guide/writing/style-guides.md]: uh. yeah. Also not sure this is the right list of style guide recommendations here.

File: docs/guide/writing/docs-principles.rst [hunk 5]
```diff
@@ -201,9 +201,9 @@ Consistent
 
 The more content editors you have, the more important a `style guide`_
 becomes in facilitating consistency. Consistency also helps make documentation
-`skimmable <#skimmable>`__ and `beautiful <#beautiful>`__
+`skimmable <#skimmable>`__ and `beautiful <#beautiful>`__.
```
Comment (Bradamant3) [inline docs/guide/writing/docs-principles.rst:205]: Perhaps add something about how a style guide can help even a single writer adhere consistent rules more efficiently?
s/content editors/writers

File: docs/guide/writing/style-guides.md [hunk 6]
```diff
@@ -0,0 +1,91 @@
...
+
+If you happen to have a background in academia or journalism, you will probably be familiar the [AP stylebook](https://en.wikipedia.org/wiki/AP_Stylebook) or the [Chicago manual of style](http://www.chicagomanualofstyle.org/book/ed17/frontmatter/toc.html). Those are great resources for writing in general, particularly for grammar and syntax, but if you're reading this page chances are you are considering style in the context of technical documentation.
+
+### Technical Document
+Style guides help you write a variety of content, such as documenting the methods of an API, or presenting a overview of complex technical concepts, or focusing on how to write particular content like user manuals, release notes, or tutorials.
+
+Other style guides often focus on the reader, helping you communicate to developers, product managers, the general public, and others. They can also focus on the language itself (tone, style, grammar). Others provide best practices on how to manage your content, version control, or publication and delivery strategies.
+
```
Comment (Bradamant3) [inline docs/guide/writing/style-guides.md]: I'm not sure I understand the distinction(s) that these two paragraphs make. The last sentence about content management, version control, etc also doesn't seem to have much if anything to do with style guides.

File: docs/guide/writing/style-guides.md (hunk 1 above)
Comment (Bradamant3) [reply docs/guide/writing/style-guides.md]: Agreed. Let's not make a recommendation here.

File: docs/guide/writing/style-guides.md (hunk 3 above)
Comment (Bradamant3) [reply docs/guide/writing/style-guides.md]: Recommend removing. There are too many ways to produce API refdoc, and this one assumes a particular workflow that is unlikely to apply, especially to the beginners for whom this content is presumably most useful.

File: docs/guide/writing/style-guides.md (hunk 4 above)
Comment (Bradamant3) [reply docs/guide/writing/style-guides.md]: Recommend removing. It's very basic content, not helpful in this context.

File: docs/guide/writing/style-guides.md [hunk 7]
```diff
@@ -0,0 +1,91 @@
...
+- [Gather Content — Developing a Content style guide](https://blog.gathercontent.com/tone-of-voice-guide)
+- [HubSpot — How to Create a Writing style guide Built for the Web](http://blog.hubspot.com/blog/tabid/6307/bid/31247/The-Simple-Template-for-a-Thorough-Content-Style-Guide.aspx)
+- [Meet Content — Editorial Style for the Web](http://meetcontent.com/blog/elements-of-editorial-style-for-the-web/)
+- [Pebble Road — Creating and maintaining a web style guide](http://www.pebbleroad.com/articles/Creating-Maintaining-a-Web-Style-Guide)
+- [Pressly — 5 style guides for better content](http://blog.pressly.com/2013/09/24/5-style-guides-craft-content/)
+- [Stanford — Creating a web style guide](https://swsblog.stanford.edu/blog/creating-web-style-guide-cardinal-work)
+- [Techwhirl – Developing a Style Guide for Technical Publications](http://techwhirl.com/developing-a-departmental-style-guide/)
+- [UCR — Writing for the Web, Content Guidelines](http://cms.ucr.edu/writing_web_content.html)
+- [Unbound — 7 things to put into a web style guide](http://unboundmedia.com/going-digital-5-key-things-to-put-into-a-style-guide-for-web-content)
```
Comment (Bradamant3) [inline docs/guide/writing/style-guides.md:92]: There are lists of resources in multiple sections here. Recommend consolidating into a single section with sub-headings that explain the differences between different lists. Or, if the differences aren't strong, provide a single list only.

File: docs/guide/writing/style-guides.md (hunk 6 above)
Comment (codeharmonics) [reply docs/guide/writing/style-guides.md]: Reordered content, made clear difference, kept the last paragraph, but made it clear that documentation organization is not what we are discussing here. I still think, however, that it is important to mention.

File: docs/guide/writing/style-guides.md (hunk 1 above)
Comment (codeharmonics) [reply docs/guide/writing/style-guides.md]: removed

File: docs/guide/writing/style-guides.md (hunk 2 above)
Comment (codeharmonics) [reply docs/guide/writing/style-guides.md]: Created a "more info" header

File: docs/guide/writing/style-guides.md (hunk 3 above)
Comment (codeharmonics) [reply docs/guide/writing/style-guides.md]: removed

File: docs/guide/writing/style-guides.md (hunk 4 above)
Comment (codeharmonics) [reply docs/guide/writing/style-guides.md]: removed

File: docs/guide/writing/style-guides.md (hunk 7 above)
Comment (codeharmonics) [reply docs/guide/writing/style-guides.md:92]: kept - I don't know where these links come from, but many of them are really good. Not sure what to do. If you really want, I'll remove. But I don't think they are easily categorized. Probably best as a single list or just removed.

**Comment by plaindocs** [comment]
Thanks @codeharmonics and @Bradamant3 -- I'm gonna merge this in for now, further iterations and improvements are of course welcome! :smiling_imp:

**Review by plaindocs — APPROVED** [review]

**Pull request merged**